standalone-metastore/metastore-rest-catalog/src/main/java/org/apache/iceberg/rest/HMSCatalogFactory.java

```diff
@@ -117,7 +118,8 @@ protected Catalog createCatalog() {
    * @return the servlet
    */
   protected HttpServlet createServlet(Catalog catalog) {
-    ServletSecurity security = new ServletSecurity(configuration);
+    String auth = MetastoreConf.getVar(configuration, ConfVars.ICEBERG_CATALOG_SERVLET_AUTH);
```

Comment:
do we even need `ICEBERG_CATALOG_SERVLET_AUTH`config, it's only set in tests?
````
  public static boolean isAuthJwt(Configuration configuration) {
    String auth = MetastoreConf.getVar(configuration, MetastoreConf.ConfVars.PROPERTIES_SERVLET_AUTH);
    return "jwt".equalsIgnoreCase(auth);
  }
````

Reply:
I think so. I want to add a new auth mode, NONE. Some query engines support neither SIMPLE nor JWT. JWT is hard to prepare for PoC

Reply:
Can't we extend PROPERTIES_SERVLET_AUTH and use it?

Reply:
Just clarifying. Do you propose using PROPERTIES_SERVLET_AUTH for Iceberg REST Catalog Servlet? I honestly don't know when we should use PropertyServlet, and I assume it is a different endpoint from Iceberg REST Catalog. Users of Iceberg REST are other computing engines, and those of PropertyServlet don't seem so. So, a separate parameter makes sense to me.

Reply:
sorry, my bad, I thought we had a generic SERVLET_AUTH config.